Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -248,6 +272,20 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::UpdateToolsMenu(bool emulation_started)
+{
+  const bool enable_wii_tools = !emulation_started || !Settings::Instance().IsWiiGameRunning();
+  m_perform_online_update_menu->setEnabled(enable_wii_tools);
+  if (enable_wii_tools)
+  {
+    IOS::HLE::Kernel ios;
+    const auto tmd = ios.GetES()->FindInstalledTMD(0x0000000100000002);
```

Comment:
Turns out there's already a constant for the system menu title ID, but it's hidden in NandPaths.h. Should I make a new header, let's say Core/CommonTitleIDs.h with all of these title IDs?